tmail-backend/mailbox/team-mailboxes/src/main/scala/com/linagora/tmail/team/TeamMailboxUserEntityValidator.scala

```diff
@@ -0,0 +1,36 @@
+package com.linagora.tmail.team
+
+import com.linagora.tmail.team.TeamMailboxUserEntityValidator.{TEAM_MAILBOX, TEAM_MAILBOX_NAMESPACE}
+import org.apache.james.core.Username
+import org.apache.james.mailbox.{MailboxManager, MailboxSession}
+import org.apache.james.mailbox.model.{MailboxConstants, MailboxPath}
+import org.apache.james.mailbox.store.MailboxReactorUtils.block
+import org.apache.james.webadmin.service.UserEntityValidator
+import org.apache.james.webadmin.service.UserEntityValidator.{EntityType, ValidationFailure}
+
+import java.util
+import javax.inject.Inject
+
+object TeamMailboxUserEntityValidator {
+  val TEAM_MAILBOX: EntityType = new EntityType("team-mailbox")
+
+  val TEAM_MAILBOX_NAMESPACE: String = MailboxConstants.NAMESPACE_PREFIX_CHAR + "TeamMailbox"
+}
+
+class TeamMailboxUserEntityValidator @Inject()(mailboxManager: MailboxManager) extends UserEntityValidator {
```

Comment:
You should use the API written by Tung...

Reply:
I know... API that is not available yet... just missed out that part when started writing yesterday